examples/demo-localstack/atmos.yaml

```diff
@@ -34,16 +34,16 @@ commands:
   - atmos validate stacks
   # Test dev stack
   - atmos terraform plan demo -s dev
-  - atmos terraform apply demo -s dev -auto-approve
-  - atmos terraform destroy demo -s dev -auto-approve
+  - atmos terraform apply demo -s dev --auto-approve
+  - atmos terraform destroy demo -s dev --auto-approve
   # Test staging stack
   - atmos terraform plan demo -s staging
-  - atmos terraform apply demo -s staging -auto-approve
-  - atmos terraform destroy demo -s staging -auto-approve
+  - atmos terraform apply demo -s staging --auto-approve
+  - atmos terraform destroy demo -s staging --auto-approve
   # Test prod stack
   - atmos terraform plan demo -s prod
-  - atmos terraform apply demo -s prod -auto-approve
-  - atmos terraform destroy demo -s prod -auto-approve
+  - atmos terraform apply demo -s prod --auto-approve
+  - atmos terraform destroy demo -s prod --auto-approve
 
```

Comment:
This seems wrong to me.

Terraform uses `-auto-approve`, not `--auto-approve`

Reply:
At a minimum, this will break existing implementations.

Reply:
Done